src/templates/snek/gql/tasks/general.ts

```diff
@@ -0,0 +1,85 @@
+//#region > Imports
+//> Parent Task
+// Contains the SNEK parent task
+import SnekTasks from "./index";
+//> Types
+// Contains the type declaration for apollo results
+import { ApolloResult } from "./index";
+//#endregion
+
+//#region > Interfaces
+/**
+ * @interface GitlabServerData defines the structure of the Gitlab server result
+ *                             data.
+ */
+interface GitlabServerData {
+  page: {
+    supportedGitlabs: [];
+  };
+}
+
+/**
+ * @interface AllPageUrlData defines the structure of the all page url result
+ *                           data.
+ */
+interface AllPageUrlData {
+  pages: [];
+}
+//#endregion
+
+//#region > Classes
+/** @class A set of session aware tasks */
+class SnekGqlGeneralTasks {
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} parent The parent task
+   */
```

Comment:
Should have some sort of comment.

?

Reply:
!

What do you mean? xD

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3